packages/cleanup-bm.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+# Copyright 2024 Edgeless Systems GmbH
+# SPDX-License-Identifier: AGPL-3.0-only
+
+set -euo pipefail
+
+echo "Starting cleanup"
+
+mapfile -t runtimeClasses < <(kubectl get runtimeclass -o jsonpath='{.items[*].metadata.name}' | tr ' ' '\n' | grep '^contrast-cc')
+
+declare -a usedRuntimeClasses=()
+declare -a unusedRuntimeClasses=()
+
+resourcesToCheck=(
```

Comment:
If we checked for an existing node installer, we could tell whether a runtime is just about to be installed.

Reply:
Is there any way of associating a given node installer with a runtime handler, so we can mark this handler as used?

Reply:
Given a `RuntimeClass`, you'd need to check the args of the node-installer containers. We could consider making this relationship easier to discover, e.g. by adding a label.